README.md

```diff
@@ -1379,6 +1379,13 @@ _Related: [Monitoring](#monitoring)_
 - [Yacy](https://yacy.net/en/index.html) - Peer based, decentralized search engine server. ([Demo](https://search.yacy.net/), [Source Code](https://github.com/yacy/yacy_search_server)) `GPL-2.0` `Java`
 
 
+### Single Sign-On (SSO)
+
+- [Kratos](https://www.ory.sh/kratos/) - Next-gen identity server with Ory-hardened authentication, MFA, FIDO2, profile management, identity schemas, social sign in, registration, account recovery, and IoT auth. Golang, headless, API-only - without templating or theming headaches. ([QuickStart](https://www.ory.sh/kratos/docs/quickstart/), [Source Code](https://github.com/ory/kratos)) `Apache-2.0` `Golang`
+- [PWM](https://github.com/pwm-project/pwm) - An open source password self-service application for LDAP directories ([Source Code](https://github.com/pwm-project/pwm)) `Apache-2.0` `Java`
```

Comment:
This one does not look like a tools for SSO, but simply a tool to change passwords in an existing ldap tree.

Reply:
Afaik it's multifunctional where i would describe it's main function as SSO, see the quickstart https://www.ory.sh/kratos/docs/quickstart/

Btw. sorrt for a long delay i didn't get notification about your message